src/simulation.c

```diff
@@ -109,7 +112,7 @@ bool step(Simulation *sp) {
     sp->time += dt;
 
     // record reaction
-    insert_reaction(sp->history,next_reaction);
+    insert(sp->history, next_reaction, dt);
```

Comment:
Do we want to record the change in time or the total time? `insert(sp->history, next_reaction, sp->time);` feels more natural to me (especially if you move the insertion above the time update)

Reply:
I don't have a strong preference here. Interconverting between the two would be easy.